### DISCUSSION TITLE
Cannot down load the GTFS data of WMATA

### DISCUSSION THREAD

**Discussion by YihengQian** [transitland/transitland#417] [Q&A]
Hello,

I can't download the GTFS data of WMATA. I've tried the URL in my browser:

https://transit.land/api/v2/rest/feed_versions/6d8827f552f7a6dd226e3a3ef221ee208479aeab/download?apikey=your-api-key

I got the error message:

{"error":"Unauthorized"}

I have a 'Hobbyist/Academic' API key which is active.

Thank you for your time. I appreciate your help.

**Comment by jworth-sm** [comment]
I'm experiencing the same thing. I have a professional API key and am attempting to download this file:

https://transit.land/api/v2/rest/feed_versions/4667594f83a4c01b1fb86bd544a7eb57828523db/download?apikey={api-key}

**Reply by drewda** [reply]
@jworth-sm please go to https://app.interline.io/users/me and click on the link reading "Click here to submit a support request" under your Transitland Professional order. Alternatively email a brief note from the account associated with the API key to support@interline.io Thank you.

**Comment by drewda** [comment]
@YihengQian the account we can see associated with your name is on the Transitland Free plan. If you are a hobbyist or an academic researcher, you may email info@interline.io to describe your project and to request an upgrade to the Hobbyist/Academic Plan.

Or if you are using an API key that is under a different name is on the Hobbyist/Academic plan, please email support@interline.io from the associated email address and include your API key so that we can investigate further.
